datafusion/core/tests/sql/select.rs

```diff
@@ -251,3 +251,63 @@ async fn test_parameter_invalid_types() -> Result<()> {
 );
     Ok(())
 }
+
+#[tokio::test]
+async fn test_parameter_empty_table_with_param_values() -> Result<()> {
```

Comment:
Given these tests aren't SQL but use the DataFrame API they probably belong in https://github.com/apache/arrow-datafusion/tree/main/datafusion/core/tests/dataframe instead